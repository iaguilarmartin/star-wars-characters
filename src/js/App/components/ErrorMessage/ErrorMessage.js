import React from 'react';

const ErrorMessage = () => (
  <div>
    <img alt="" src="/assets/images/porg.png" />
  </div>
);

export default ErrorMessage;
