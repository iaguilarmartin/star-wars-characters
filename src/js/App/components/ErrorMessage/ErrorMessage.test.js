import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import ErrorMessage from './ErrorMessage';

describe('<ErrorMessage/>', () => {
  it('renders', () => {
    shallow(<ErrorMessage />);
  });

  it('matches the snapshot', () => {
    const wrapper = shallow(<ErrorMessage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
