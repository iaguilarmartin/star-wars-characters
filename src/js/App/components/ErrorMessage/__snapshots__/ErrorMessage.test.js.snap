// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage/> matches the snapshot 1`] = `
<Section>
  <Image
    alt=""
    src="/assets/images/porg.png"
  />
  ooopss something went wrong...
</Section>
`;
